src/renderer/components/MainPage.tsx

```diff
@@ -161,7 +177,21 @@ class MainPage extends React.PureComponent<Props, State> {
         this.setState({tabViewStatus: status});
     }
 
+    async requestDownloadsLength() {
+        try {
+            const count = await window.ipcRenderer.invoke(REQUEST_HAS_DOWNLOADS);
+            this.setState({
+                hasDownloads: count,
+            });
+        } catch (error) {
+            console.error(error);
```

Comment:
Is there any case that this will popup on load? We've had that happen in the past and I'm trying to avoid pointless error messages cause they confuse people.